packages/core/src/view/util.ts

```diff
@@ -230,7 +230,10 @@ export function getParentRenderElement(view: ViewData, renderHost: any, def: Nod
     if ((renderParent.flags & NodeFlags.TypeElement) === 0 ||
         (renderParent.flags & NodeFlags.ComponentView) === 0 ||
         (renderParent.element!.componentRendererType &&
-         renderParent.element!.componentRendererType!.encapsulation === ViewEncapsulation.Native)) {
+         (renderParent.element!.componentRendererType!.encapsulation ===
+              ViewEncapsulation.ShadowDom ||
+          // TODO: remove the Native fallback logic in v12.
+          renderParent.element!.componentRendererType!.encapsulation === 1))) {
```

Comment:
I'd propose to introduce an internal constant to refer to the deprecated view encapsulation value, so that it's easier to find and remove all usages in v12 (and also keep the `TODO` comment):
```suggestion
          renderParent.element!.componentRendererType!.encapsulation === ɵDeprecatedNativeViewEncapsulation))) {
```

What do you think?

Reply:
My reasoning behind not having a constant is that we only use it in a couple of places in two different packages (`core` and `platform-browser`). Adding an exposing the constant would be a lot of boilerplate for not much benefit.

Reply:
Got it, thanks for the comment. I guess in this case we can introduce separate constants (with the same name) in two packages (so we don't have to link them together). The main benefit is to have an easy way to find all references later (and it'd be convenient to use a symbol for it), so we can clean them up. Another approach might be to add an inline comment like this:

```
          // TODO: remove the Native fallback logic in v12.
          renderParent.element!.componentRendererType!.encapsulation === 1 /* deprecated ViewEncapsulation.Native */))) {
```

Reply:
I left a comment next to them, but it might be best to create a Jira task and link it in the comment. I'll do that when addressing the rest of the feedback.

Reply:
Ok, thanks 👍